lib/nerdtree/bookmark.vim

```diff
@@ -288,7 +288,7 @@ function! s:Bookmark.str()
     let pathStr = self.path.str({'format': 'UI'})
     if strdisplaywidth(pathStr) > pathStrMaxLen
         while strdisplaywidth(pathStr) > pathStrMaxLen && strchars(pathStr) > 0
-            let pathStr = strcharpart(pathStr, 1)
+            let pathStr = substitute(pathStr, '.\{1}', '', '')
```

Comment:
I think the regex `'^.'` would be safer to use here. That guarantees it finds the first character.